MatrixSDKTests/TestPlans/UnitTestsWithSanitizers.xctestplan

```diff
@@ -45,6 +45,10 @@
         "MXAsyncTaskQueueUnitTests",
         "MXAuthenticationSessionUnitTests",
         "MXBackgroundTaskUnitTests",
+        "MXBeaconInfoUnitTests\/testJSONDictionaryFail()",
```

Comment:
Afaik we prefer to edit this file manually and add only the class name, instead of every method of it.